### PR TITLE
Create a data field with story in storybook

### PR DESCRIPTION
**Pull request by meloneminze**
I create a div with an input field and a textarea where the user can enter the informations about their children. In the future i will create a dynamic textarea as a feature. 

image

close #25

### CODE REVIEW

File: client/src/components/SimpleDataField.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  outline: none;
+`;
+
+const Headline = styled.input`
+background-color: ${(props) => props.theme.colors.input};
+color: ${(props) => props.theme.colors.textSecondary}
+font-weight: bold;
+outline: none;
+cursor: pointer;
+max-width: 100%;
```
Comment (hauptdigital) [inline client/src/components/SimpleDataField.js:21]: Currently, the `size={65}` is setting a fix width bigger than your `Textareacontainer` for the input but the `max-width: 100%` will then reduce it to fit into the Container. The same happens with the textarea. 

The suggestions is to remove the fix width properties like `size={65}` from the input and `cols={60}` from the textarea and replace 'max-width: 100%' with `width: 100%`

```suggestion
width: 100%;
```

File: client/src/components/SimpleDataField.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    color: ${(props) => props.theme.colors.textSecondary};
+`;
+
+const Textarea = styled.textarea`
+background-color: ${(props) => props.theme.colors.input};
+  color: ${(props) => props.theme.colors.textPrimary};
+  border: none;
+  outline: none;
+  cursor: pointer;
+  max-width: 100%;
```
Comment (hauptdigital) [inline client/src/components/SimpleDataField.js:34]:
```suggestion
  width: 100%;
```

File: client/src/components/SimpleDataField.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+`;
+
+const SimpleDataField = () => {
+  return (
+    <TextareaContainer>
+      <Headline
+        placeholder="Adresse der Schule"
+        minLength={1}
+        maxLength={50}
+        size="65"
```
Comment (hauptdigital) [inline client/src/components/SimpleDataField.js:48]:
```suggestion
```

File: client/src/components/SimpleDataField.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      ></Headline>
+      <Textarea
+        placeholder="Aquariumstraße 42"
+        maxLength={250}
+        rows={3}
+        cols={60}
```
Comment (hauptdigital) [inline client/src/components/SimpleDataField.js:54]:
```suggestion
```

File: client/src/components/SimpleDataField.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+import styled from '@emotion/styled';
+
+const TextareaContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: ${(props) => props.theme.colors.input};
+  border-radius: 1.25rem;
+  border: none;
+  padding: 0.6rem;
```
Comment (hauptdigital) [inline client/src/components/SimpleDataField.js:11]: Just a personal opinion to add a bit more padding here:

```suggestion
  padding: 1rem;
```

File: client/src/components/SimpleDataField.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const TextareaContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```
Comment (marwinburesch) [inline client/src/components/SimpleDataField.js:7]: you can use the shorthand `flex-flow` here

File: client/src/components/SimpleDataField.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  outline: none;
+  cursor: pointer;
+  max-width: 100%;
+  resize: none;
+  wrap: "off";
+  ::placeholder {
+    color: ${(props) => props.theme.colors.textSecondary};
+`;
+
+const SimpleDataField = () => {
```
Comment (marwinburesch) [inline client/src/components/SimpleDataField.js:41]: This sounds very generic, I would name this component differently. Maybe: `MemberInfoInput`, `MemberDetailInput` or something that indicates a group of components: 'EditMemberDetail`

File: client/src/components/SimpleDataField.js (hunk 1 above)
Comment (meloneminze) [reply client/src/components/SimpleDataField.js:21]: You are absolutly right! Thanks!

File: client/src/components/SimpleDataField.js (hunk 6 above)
Comment (meloneminze) [reply client/src/components/SimpleDataField.js:7]: Of course!

File: client/src/components/SimpleDataField.js (hunk 7 above)
Comment (meloneminze) [reply client/src/components/SimpleDataField.js:41]: MemberInfoInput sounds good. I will change it.

File: client/src/components/SimpleDataField.js (hunk 5 above)
Comment (meloneminze) [reply client/src/components/SimpleDataField.js:11]: I like your personal opinion!

**Pull request merged**